src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -0,0 +1,245 @@
+from datetime import datetime
+import json
+import os
+import sas
+import sas_testcase
+from util import winnforum_testcase
+class FullActivityDumpTestcase(sas_testcase.SasTestCase):
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  @classmethod
+  def setUpClass(self):
+    self._sas_server = sas.GetServer()
+    self._sas_server.StartServer()
+
+  @classmethod
+  def tearDownClass(self):
+    self._sas_server.StopServer()
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_FAD_2(self):
+    # Load the Config file
+    config = loadConfig('testcases/configs/test_WINNF_FT_S_FAD_2/default.config')
```

Comment:
Please the configurability framework established in REG.11 and commit default.config.

Reply:
I followed the REG.11 to put all the load file into default.config first. However, I still need some extra config to for the future config. So in the latest code, I added extra.config for each test case. Because, for the test cases some fields need to do future configuration

Reply:
In the new code, For the test cases which we need some external parameters, I created a extra.config file. The extra.config file will store all the external parameters which can be easily to tun in the future. Then we could load the external parameters and work with the other parameters we have in testdata folder to generate parameter we have to put into the default.config. I think this will be a better way